content/docs/2.5/scalers/gcp-pub-sub.md

```diff
@@ -29,14 +29,15 @@ The `credentialsFromEnv` property maps to the name of an environment variable in
 Here's an [example](https://github.com/kedacore/sample-go-gcppubsub).
 
 ### Authentication Parameters
-You can use `TriggerAuthentication` CRD to configure the authenticate by providing the service account credentials in JSON. 
+There are three options to configure authentication:
```

Comment:
You mention three, but I only see two?

Reply:
Oh you mean the sub items of the TA CRD?

Reply:
Jup, there are 2 axis, either inline or with TriggerAuthentication. And within triggers there are another 2 options, which differ even more than whether you break out the authentication to a secret or not: using an SA saves you from managing the access credentials all together.